tests/integration/test_tls.py

```diff
@@ -34,7 +38,6 @@
 DUMMY_NAME = "app"
 
 
-@pytest.mark.abort_on_fail
```

Comment:
**todo**: Can we add a FIXME to add back the decorator?

Reply:
Actually, I realise that I didn't want this, and should have done `wait_for_idle(raise_on_error=False)`. I'll make that change now.